toolkit/ar/src/main/java/com/arcgismaps/toolkit/ar/WorldScaleSceneView.kt

```diff
@@ -116,197 +114,197 @@ public fun WorldScaleSceneView(
+                visualizePlanes = false
+            )
+            // Once the session is created, we can say we're initialized
+            initializationStatus.update(
+                WorldScaleSceneViewStatus.Initialized,
+                onInitializationStatusChanged
+            )
+        }
+        SceneView(
+            arcGISScene = arcGISScene,
+            modifier = Modifier.fillMaxSize(),
+            onViewpointChangedForCenterAndScale = onViewpointChangedForCenterAndScale,
+            onViewpointChangedForBoundingGeometry = onViewpointChangedForBoundingGeometry,
+            graphicsOverlays = graphicsOverlays,
+            sceneViewProxy = worldScaleSceneViewProxy.sceneViewProxy,
+            sceneViewInteractionOptions = remember {
+                // Disable interaction, which is not supported in WorldScale scenarios
+                SceneViewInteractionOptions().apply {
+                    isEnabled = false
                 }
+            },
+            viewLabelProperties = viewLabelProperties,
+            selectionProperties = selectionProperties,
+            isAttributionBarVisible = isAttributionBarVisible,
+            onAttributionTextChanged = onAttributionTextChanged,
+            onAttributionBarLayoutChanged = onAttributionBarLayoutChanged,
+            cameraController = cameraController,
+            analysisOverlays = analysisOverlays,
+            imageOverlays = imageOverlays,
+            atmosphereEffect = AtmosphereEffect.None,
+            timeExtent = timeExtent,
+            onTimeExtentChanged = onTimeExtentChanged,
+            spaceEffect = SpaceEffect.Transparent,
+            sunTime = sunTime,
+            sunLighting = sunLighting,
+            ambientLightColor = ambientLightColor,
+            onNavigationChanged = onNavigationChanged,
+            onSpatialReferenceChanged = onSpatialReferenceChanged,
+            onLayerViewStateChanged = onLayerViewStateChanged,
+            onInteractingChanged = onInteractingChanged,
+            onCurrentViewpointCameraChanged = onCurrentViewpointCameraChanged,
+            onRotate = onRotate,
+            onScale = onScale,
+            onUp = onUp,
+            onDown = onDown,
+            onSingleTapConfirmed = onSingleTapConfirmed,
+            onDoubleTap = onDoubleTap,
+            onLongPress = onLongPress,
+            onTwoPointerTap = onTwoPointerTap,
+            onPan = onPan,
+            content = {
+                content?.invoke(WorldScaleSceneViewScope(this))
             }
-        }
+        )
+    }
+}
+
+/**
+ * Starts the a [SystemLocationDataSource] and periodically updates the [cameraController] with new locations
+ *
+ * @since 200.7.0
+ */
+@Composable
+private fun LocationTracker(
+    cameraController: TransformationMatrixCameraController
+) {
+    val locationDataSource = rememberSystemLocationDataSource().locationDataSource
+    // We should reset the origin camera if the LDS or camera controller changes
```

Comment:
Given the location data source won't change, I think this comment needs updating:
```suggestion
    // We should reset the origin camera if the camera controller changes
```